marconi-sidechain/src/Marconi/Sidechain/Bootstrap.hs

```diff
@@ -70,21 +84,22 @@ runSidechainIndexers = do
           , Just $ dbPath </> utxoDbName
           )
         ,
-          ( epochStateWorker (CLI.nodeConfigPath cliArgs) epochStateCallback
+          ( epochStateWorker nodeConfigPath epochStateCallback
           , Just $ dbPath </> epochStateDbName
           )
         ,
-          ( mintBurnWorker mintBurnCallback $ CLI.targetAssets cliArgs
+          ( mintBurnWorker mintBurnCallback targetAssets
           , Just $ dbPath </> mintBurnDbName
           )
         ]
+
   lift $
     runIndexers
       trace
-      (CLI.optionsRetryConfig cliArgs)
-      (CLI.socketFilePath cliArgs)
-      (CLI.networkId cliArgs)
+      optionsRetryConfig
+      socketFilePath
+      networkId
       C.ChainPointAtGenesis
-      (CLI.minIndexingDepth cliArgs)
-      (CLI.optionsFailsIfResync cliArgs)
+      minIndexingDepth
+      optionsFailsIfResync
```

Comment:
Rationale for removing the qualified import?

Reply:
The changes in this file aren't needed for the ticket, but the file was changed for the previous ticket and I thought I'd tidy things up while I was still here. I think it's much more readable with the `RecordWildCards` instead of the qualified import and explicit record accessors, but I realize that may not fit with the coding style in the rest of the project. If you're not in favour, I'll happily remove these changes.

Reply:
I see your point. In this case, I felt it was OK because the fields are all named explicitly in the import, so if you want to know where one of the names comes from (as you would even with a local variable) a search will find it immediately in the import. However, that's still not as immediate as seeing the qualification right at the point of use.

I'll go ahead and revert these changes.